### DISCUSSION TITLE
What is the significance of `ResourceBuilderExtensions.ExcludeFromManifest`?

### DISCUSSION THREAD

**Discussion by peteraritchie** [microsoft/aspire#1047] [Q&A] [answered]
What is the significance of `ResourceBuilderExtensions.ExcludeFromManifest`?  If a manifest is a description of relationships between distributed dependencies for tool developers to build upon for deployment, why would I not want a tool to know about a dependent resource?  I'd expect that a tool that did not know about a particular type of resource to warn or halt upon something it could not support.

**Comment by DamianEdwards** [comment, marked as answer]
It's typically for scenarios where you want a resource composed into the AppHost only for the purposes of supporting the developer inner-loop, such that it should never be reported as an actual app resource outside of that context.
